drivers/eeprom/i2c_xx24xx.c

```diff
@@ -253,7 +275,8 @@ static int ee24xx_waitwritecomplete(FAR struct ee24xx_dev_s *eedev,
   uint32_t addr_hi = (memaddr >> (eedev->addrlen << 3));
 
   msgs[0].frequency = eedev->freq;
-  msgs[0].addr      = eedev->addr | (addr_hi & ((1 << eedev->haddrbits) - 1));
+  msgs[0].addr      = eedev->addr | \
+                      (addr_hi & ((1 << eedev->haddrbits) - 1));
```

Comment:
The line continuation character \ should not be used in C code.  It is not necessary in C.